Packs/GCP/Integrations/GCP/GCP.py

```diff
@@ -256,6 +260,66 @@ def extract_zone_name(zone_input: str | None) -> str:
     return zone_input.strip()
 
 
+def parse_labels(labels_str: str) -> dict:
+    """
+    Transforms a string of multiple inputs to a dictionary
+    Args:
+        labels_str (str): The actual string to parse to a key & value pair.
+
+    Returns:
+        Returns the labels dictionary with the extracted key & value pairs.
+    """
+    labels = {}
+    for f in labels_str.strip().split(";"):
+        if f:
+            match = KEY_VALUE_ITEM_REGEX.match(f)
+            if match is None:
+                raise ValueError(
+                    f"Could not parse field: {f}. Please make sure you provided like so: key=abc,value=123;key=def,value=456"
+                )
+
+            labels.update({match.group(1).lower(): match.group(2).lower()})
+    return labels
+
+
+def handle_permission_error(e: HttpError, project_id: str, command_name: str):
+    """
+    Given an error, extract the relevant information (account_id & message & permission name) to report to the backend.
+    Args:
+        e (HttpError): The error object.
+        project_id (str): The project identifier.
+        command_name (str): The name of the command that was executed.
+
+    Returns:
+        Returns the labels dictionary with the extracted key & value pairs.
+    """
+    demisto.debug(f"{e.resp=}")
+    status_code = e.resp.status
+    if e.resp.get("content-type", "").startswith("application/json"):
```

Comment:
Is there a specific error code that we will get in this case? 403?

Reply:
It can be 403 or 401